fdbserver/DDShardTracker.actor.cpp

```diff
@@ -208,6 +211,10 @@ int64_t getMaxShardSize(double dbSizeEstimate) {
 	                (int64_t)SERVER_KNOBS->MAX_SHARD_BYTES);
 }
 
+bool ddLargeTeamEnabled() {
```

Comment:
This function can be const.

Reply:
since this function is not within a class or struct it cannot be const